test/test_query.cpp

```diff
@@ -4107,6 +4107,46 @@ TEST(Query_LinkChainSortErrors)
     CHECK_LOGIC_ERROR(t1->get_sorted_view(SortDescriptor({{t1_linklist_col}})), ErrorCodes::InvalidSortDescriptor);
 }
 
+TEST_TYPES(Query_SortingCompressedStrings, std::true_type, std::false_type)
```

Comment:
This test is covering very little of the newly added code. It needs to check sorting mixed with all of compressed strings, non-compressed strings, and non-strings, along with sorting over links.

Reply:
Yep you are right, but we do have many of these tests already, I will add more tests like this and be sure we are covering all, and not solely rely on our existing tests (if they commit the strings before)

Reply:
I have updated the tests in order to cover what you asked @tgoyne .